react-with-graphql-tutorial.md

```diff
@@ -936,11 +937,11 @@ const Organization = ({ organization, errors }) => {
 };
 {{< /highlight >}}
 
-Congratulations, you performed your first GraphQL query in a React application. The great part about it: There was nothing magical about it. It was a plain HTTP POST request with a query as payload. You have used a configured axios client to perform request with a HTTP POST method. Afterward, you were able to store the result in React's local state in order to display it eventually.
+You performed your first GraphQL query in a React application, a plain HTTP POST request with a query as payload. You have used a configured axios client to perform request with a HTTP POST method. Afterward, you were able to store the result in React's local state to display it later.
```

Comment:
"You have used a configured axios client to perform request with a HTTP POST method." to "You have used a configured axios client instance for it."

Reply:
Maybe I would leave "The great part about it: There was nothing magical about it. It was a plain HTTP POST request with a query as payload." in there.

Reply:
"The great part about it: There was nothing magical about it"   Not a good sentence.  Unnecessary colon in the middle. Use of words "great" and "magical" (two overused, exaggerative words in English).  The point is made in the next sentence, the fact it's just a plain HTTP POST and nothing special, so you can just make that point without the introductory sentence that doesn't add any value.